mods/player_api/api.lua

```diff
@@ -125,12 +132,7 @@ minetest.register_globalstep(function()
 			-- Apply animations based on what the player is doing
 			if player:get_hp() == 0 then
 				player_set_animation(player, "lay")
-			-- Determine if the player is walking
 			elseif controls.up or controls.down or controls.left or controls.right then
-				if player_sneak[name] ~= controls.sneak then
-					player_anim[name] = nil
-					player_sneak[name] = controls.sneak
-				end
```

Comment:
Why is this no longer needed?
I admit i actually do not understand why this code block is needed.

Reply:
Ah, perhaps this is what you mean by 'Removes need for workaround for changing animation speed' ?
In https://github.com/minetest/minetest_game/commit/4c0c7ae146bb57f56eb03e804853db73d31ed957#diff-ee5a751279baa5e37734fc0c1e75daf3 the old comment for this was:
`-- Refresh player animation below if sneak state changed`

Reply:
> Ah, perhaps this is what you mean by 'Removes need for workaround for changing animation speed' ?

Yes.